src/solver/conv_asm_1x1u_bias_activ_fused.cpp

```diff
@@ -208,6 +207,15 @@ ConvBiasActivAsm1x1U::GetSolution(const FusionContext& context,
     return sol;
 }
 
+float ConvBiasActivAsm1x1U::GetWti(const FusionContext&, const FusionDescription&) const
+{
+    // Negative values mean rough estimation of time
+    // Solvers with positive values are considered first priority
+    // If none found solvers with negative are used, -2.0f meaning a completely unknown value
+    // So lower absolute values mean higher priority
+    return -.5f;
```

Comment:
[Quality] Return `wti_unknown / 4` here. This provides a symbolic connection from this -0.5 to -2.0.

Reply:
To be fair I think `unknown / 4` is confusing :)

Reply:
Let's invent some other name. E.g. `wti_unknown_or_worst` or something else.